spec/index.html

```diff
@@ -5669,248 +5670,267 @@ <h5>NOT IN</h5>
             </ul>
+            
+            <div class="result">
+              <table>
+                <tbody>
+                  <tr>
+                    <td><code>sameTerm(&lt;http://example/&gt;, &lt;http://example/&gt;)</code></td>
+                    <td>true</td>
+                  </tr>
+                  <tr>
+                    <td><code>sameTerm(&lt;http://example/&gt;, &lt;https://example/&gt;)</code></td>
+                    <td>false</td>
+                  </tr>
+                  <tr>
+                    <td><code>sameTerm("abc", "abc")</code></td>
+                    <td>true</td>
+                  </tr>
+                  <tr>
+                    <td><code>sameTerm("abc"@en, "abc")</code></td>
+                    <td>false</td>
+                  </tr>
+                  <tr>
+                    <td><code>sameTerm("abc"@en, "abc"@EN)</code></td>
+                    <td>true</td>
+                  </tr>
+                  <tr>
+                    <td><code>sameTerm("abc"@en--rtl, "abc"@en)</code></td>
+                    <td>true</td>
+                  </tr>
+                  <tr>
+                    <td><code>sameTerm(2, 2.0)</code></td>
+                    <td>false</td>
+                  </tr>
+                  <tr>
+                    <td><code>sameTerm(2, "2"^^xsd:integer)</code></td>
+                    <td>true</td>
+                  </tr>
+                  <tr>
+                    <td><code>sameTerm(2, "02"^^xsd:integer)</code></td>
+                    <td>false</td>
+                  </tr>
+              </table>
+            </div>
+
+          </section>
+          <section id="func-RDFterm-equal">
+            <h5>RDFterm-equal</h5>
+            <pre class="prototype nohighlight">
+              <span class="return">xsd:boolean</span> <span class="operator" style="text-transform: none;">RDFterm-equal</span> (<span class="type"><span class="type RDFterm">RDF term</span></span> <span class="name">term1</span>, <span class="type"><span class="type RDFterm">RDF term</span></span> <span class="name">term2</span>)
+            </pre>
+
+            <p>This function cannot be used directly in expressions. The purpose
+              of this function is to define the semantics of the "=" operator when applied to
+              two RDF terms that do not fall into any of the other, more concrete cases
+              covered in the operator mapping table in Section
+              <a href="#OperatorMapping" class="sectionRef"></a>.
+            </p>
+
+            <p class="ednote" style="background-color: #EEE">
+              This will be renamed <code>sameValue</code></p>
+
+            <ol>
+              <li>If one or both arguments are known to be ill-typed, 
+                then produce a type error.
+              </li>
+              <li>Returns TRUE if <code>term1</code> and <code>term2</code>
+                are <em>equal</em> RDF terms, as defined below.
+              </li>
```

Comment:
There is no such definition of "_equal_ RDF terms" below.

Reply:
RDF Concepts has caught up.
https://w3c.github.io/rdf-concepts/spec/index.html#dfn-rdf-term-equality